.github/workflows/dev_deploy.yml

```diff
@@ -52,8 +52,10 @@ jobs:
       - name: Terraform Validate
         id: validate
         run: terraform validate -no-color
+      # Note that while we do pass a value of stela_staging_image below, it won't be used because the -target option
+      # restricts terraform to just updating the dev deployment
```

Comment:
Interesting, thanks for this. Why include the staging image then?

Reply:
Terraform won't let us leave the variable empty